BUILDING.md

```diff
@@ -156,8 +192,8 @@ explains how to install all prerequisites.
 
 #### Prerequisites
 
-* `gcc` and `g++` 4.9.4 or newer, or
-* `clang` and `clang++` 3.4.2 or newer (macOS: latest Xcode Command Line Tools)
+* `gcc` and `g++` >=6.3 or newer, or
+* `clang` and `clang++` 3.4.2 or newer (macOS: Xcode Command Line Tools >= 7)
 * Python 2.7
```

Comment:
@nodejs/python is it time to bump this to Python 3 since Python 2 EOL will hit us pretty quickly in Node 12's lifetime?

Reply:
We are nearly there. But we cannot drop Python 2.7 dependency in this release.